Src/DependencyCollector/Shared/Implementation/FrameworkHttpEventListener.cs

```diff
@@ -78,13 +78,23 @@ protected override void OnEventWritten(EventWrittenEventArgs eventData)
                 switch (eventData.EventId)
                 {
                     case BeginGetResponseEventId:
-                        this.OnBeginGetResponse(eventData);
+                        if (!DependencyTableStore.Instance.IsDesktopHttpDiagnosticSourceActivated)
```

Comment:
can you remind me - will this method check that it was actually enabled or just configured to be enabled?